packages/core/src/useFormField/useFormField.ts

```diff
@@ -191,7 +192,7 @@ export function useFormField<TValue = unknown>(opts?: Partial<FormFieldOptions<T
     form.setFieldDisabled(path, disabled);
   });
 
-  return field;
+  return { ...field, form };
```

Comment:
No need to spread, just add the `resolvedForm` to the `FormField` type and then you could just set it as a prop here.